crates/bin/pindexer/src/lqt/schema.sql

```diff
@@ -1,32 +1,179 @@
-CREATE TABLE IF NOT EXISTS lqt_votes (
-    id SERIAL PRIMARY KEY,
+CREATE SCHEMA lqt;
+
+CREATE TABLE lqt._params (
+  epoch INTEGER PRIMARY KEY,  
+  delegator_share NUMERIC(3, 2) NOT NULL,
+  gauge_threshold NUMERIC(3, 2) NOT NULL
+);
+
+CREATE TABLE lqt._finished_epochs (
+    epoch INTEGER PRIMARY KEY
+);
+
+CREATE TABLE lqt._available_rewards (
+  epoch INTEGER PRIMARY KEY,  
+  amount NUMERIC NOT NULL
+);
+
+CREATE TABLE lqt._delegator_rewards (
     epoch INTEGER NOT NULL,
-    incentivized BYTEA NOT NULL,
-    power BIGINT NOT NULL,
-    tx_id BYTEA NOT NULL,
-    rewards_recipient BYTEA NOT NULL
+    address BYTEA NOT NULL,
+    amount NUMERIC NOT NULL,
+    PRIMARY KEY (epoch, address)
 );
 
-CREATE INDEX ON lqt_votes (epoch, incentivized);
+CREATE INDEX ON lqt._delegator_rewards (address);
 
-CREATE TABLE IF NOT EXISTS lqt_delegator_rewards (
-    id SERIAL PRIMARY KEY,  
+CREATE TABLE lqt._lp_rewards (
     epoch INTEGER NOT NULL,
-    reward BIGINT NOT NULL,
-    rewards_recipieint BYTEA NOT NULL,
-    incentivized BYTEA NOT NULL
+    position_id BYTEA NOT NULL,
+    asset_id BYTEA NOT NULL,
+    amount NUMERIC NOT NULL,
+    executions INTEGER NOT NULL,
+    um_volume NUMERIC NOT NULL,
+    asset_volume NUMERIC NOT NULL,
+    um_fees NUMERIC NOT NULL,
+    asset_fees NUMERIC NOT NULL,
+    points NUMERIC NOT NULL,
+    PRIMARY KEY (epoch, position_id)
 );
 
-CREATE INDEX ON lqt_delegator_rewards (epoch);
+CREATE INDEX ON lqt._lp_rewards (asset_id);
 
-CREATE TABLE IF NOT EXISTS lqt_position_rewards (
-    id SERIAL PRIMARY KEY,  
+CREATE TABLE IF NOT EXISTS lqt._votes (
+    id SERIAL PRIMARY KEY,
     epoch INTEGER NOT NULL,
-    reward BIGINT NOT NULL,
-    position BYTEA NOT NULL,
-    incentivized BYTEA NOT NULL,
-    pair_volume NUMERIC(39) NOT NULL,
-    position_volume NUMERIC(39) NOT NULL
+    power NUMERIC NOT NULL,
+    asset_id BYTEA NOT NULL,
+    address BYTEA NOT NULL
 );
 
-CREATE INDEX ON lqt_position_rewards (epoch);
+CREATE INDEX ON lqt._votes (epoch);
+CREATE INDEX ON lqt._votes (address);
+
+CREATE VIEW lqt.summary AS
```

Comment:
I think it would be helpful to have natural language summaries for each materialized views. The query is self-explanatory but the higher level purpose of the view, e.g, for the summary of lqt votes we need:
- to display how many reward were issued in total for each epoch
- to display the staking power mobilized for the epoch
- etc.

Reply:
I added in-schema comments, which can be queried from psql.